### ISSUE TITLE
What is the API rate limit? 

### ISSUE THREAD

**Issue by samayo** [scrutinizer-ci/scrutinizer#381]
I'm trying to fetch some details about repositories from scrutinizer-ci, but it seems I can not find the rate limit as in hour many requests I can send to:   
 `https://scrutinizer-ci.com/api/repositories/g/{USER/REPO}?access_token={TOKEN}`. 

Is there any official rate limit? If, so is there a JSON response that inform you when the rate limit is exceeded?

**Comment by schmittjoh** [comment]
We currently do not enforce any limit. Anything specific you are planning?

**Comment by samayo** [comment]
Nah, just a pet project. Once it's done and public I'll reply back to this thread and let you see it instead. :smile:
